resources/hooks/local/pre-push

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+GIT_USER=$(git config user.name)
+GIT_EMAIL=$(git config user.email)
+COMMIT_MSG_FILE=$1
+
+# Fetch the GIT diff and format it as command input:
+DIFF=$(git diff -r -p -m -M --full-index --staged | cat)
```

Comment:
This will mostly result in an empty set of files since on push the staged changes are mostly empty of should not be checked since those aren't the files that you're committing. We'll need to find a way to check which files are transfered to the remote.